Source/WebCore/platform/graphics/texmap/TextureMapperPlatformLayerProxy.cpp

```diff
@@ -134,7 +140,7 @@ void TextureMapperPlatformLayerProxy::swapBuffer()
 
 void TextureMapperPlatformLayerProxy::pushNextBuffer(std::unique_ptr<CoordinatedPlatformLayerBuffer>&& newBuffer)
 {
-    ASSERT(m_lock.isHeld());
```

Comment:
You could reverse the condition to assure we didn't miss a call site?

Reply:
No need, because `lock()` is gone and `m_lock` is private.